docs/user/integration/aws-cloudwatch/aws-otel.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: otel-collector
+data:
+  collector.conf: |
+    receivers:
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: otel-collector
+              scrape_interval: 5s
+              static_configs:
+                - targets: [localhost:8888]
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        limit_mib: 50
```

Comment:
This does not make sense. We give 1 GB memory to the pod, but allow only using 50 MB. Let's use the same settings here that we have for our own Otel collectors. Same for the resource settings.